app/views/hostgroups/index.html.erb

```diff
@@ -6,6 +6,8 @@
   <thead>
     <tr>
       <th><%= sort :label, :as => _('Hostgroup|Name') %></th>
+      <th><%= sort :label, :as => _('Hosts') %></th>
+      <th><%= sort :label, :as => _('Child hosts')%></th>
```

Comment:
Yeah, perhaps.  Another suggestion is "Hosts including sub-groups" (thanks @LeoAdamek).